manta-js/examples/sdk-example/index.ts

```diff
@@ -124,27 +124,25 @@ const getBaseWallet = async () => {
       return result || null;
     },
   });
-  BaseWallet.onWasmCalledJsErrorCallback = (err, palletName) => {
```

Comment:
Wasm is calling the ledger api. If the network fails, it will trigger wasm crash and report an error; this is optimized in wasm, so this error is unnecessary